mmk_default_rules_alpha.mms

```diff
@@ -195,6 +195,47 @@ GENCAT	    = GENCAT
 GENCATFLAGS =
 
 
+!SDL_UNSUPPORTED_LANGUAGES=DTR:.? DUMP:.? UIL:.UIL?
+!SDL_LANGUAGES=ADA:.ADA BASIC:.BAS BLISS:.R32 BLISS64:.R64 CC:.C DCL:.COM -
+!	FORTRAN:.FOR MACRO:.MAR PASCAL:.PAS PLI:.PL1 SDML:.SDML -
+! 	TPU:.TPU
+!$(FOREACH LANG,$(SDL_LANGUAGES),-
+!.SDL$(LASTWORD $(PATSUBST :, ,$(LANG))) : ; -
+!$(SDL) $(SDLFLAGS)/LANG=$(FIRSTWORD $(PATSUBST :, ,$(LANG)))=$(MMS$TARGET) $(MMS$SOURCE))
```

Comment:
The above commented lines was my attempt at generating the implicit rules instead of writing them explicitely.  Unfortunately, MMKC doesn't like it:

```
$ MMKC/OUTPUT=ETC_DIR:MMK_DEFAULT_RULES.H MMK_DEFAULT_RULES_Alpha.MMS
%MMK-F-PARSERR, error parsing description line $(FOREACH LANG,$(SDL_LANGUAGES),.SDL$(LASTWORD $(PATSUBST :, ,$(LANG))) : ; $(SDL) $(SDLFLAGS)/LANG=$(FIRSTWORD $(PATSUBST :, ,$(LANG)))=$(MMS$TARGET) $(MMS$SOURCE))
-MMK-I-ERRLOC, at line number 204 in file USER:[LEVITTE.TEST.mmk-issue17-support
-SDL]mmk_default_rules_alpha.mms;5
-LIB-F-SYNTAXERR, string syntax error detected by LIB$TPARSE
```